assets/src/edit-story/components/popup/test/popup.js

```diff
@@ -34,7 +34,24 @@ describe('Popup', () => {
 
     const popup = getByText('Hello World!');
     expect(popup).toBeDefined();
-    expect(popup).toHaveAttribute('width', '270');
+  });
+
+  it('should render popup in different placements', () => {
+    const anchor = {
+      current: document.createElement('div'),
+    };
+    document.body.append(anchor.current);
+
+    Object.values(Placement).map((placement) => {
+      const { getByText } = renderWithTheme(
+        <Popup anchor={anchor} isOpen={true} placement={placement}>
+          {`Hello ${placement}!`}
+        </Popup>
+      );
+
+      const popup = getByText(`Hello ${placement}!`);
+      expect(popup).toBeDefined();
```

Comment:
Should we test something more useful here? The placement has a default and a fallback so not sure how the placement would influence the popup not being defined.

This should perhaps test the actual placement then -- if it was set correctly.

Reply:
And also the fallback -- if there is not placement set or an incorrect placement is set for some reason.

Reply:
This test was mostly a placeholder. The issue here is that layout/boundingrect functions are unavailable in the jest environment which makes it hard to test for placement. I need to think a bit more about it but since this PR is critical, I added a TODO so that we do not block on this.

Reply:
Perhaps we should remove the tests in loop to not have the deceiving 100% test coverage for that part then?

We could leave just one test as the placeholder (instead of all the placements) + the TODO to ensure the coverage reflects the actual test coverage.

E.g. a test that adding a non-existing placement value does still render the popup which is actually useful.
And then leave the other placements out for accuracy. Thoughts?

Reply:
Done :)